kaggle_environments/envs/lux_ai_2021/testing.md

```diff
@@ -0,0 +1,9 @@
+# Testing
+
+Some tests can be automated, however since the actual design is programmed in a seperate repository and uses a different engine for testing, some things needs to be manually checked at the moment
+
+Whenever the design is changed, the following tests should be done:
+
```

Comment:
There isn't a programmatic way at the moment because kaggle replay files are formatted differently than the replay files players generate locally (this can be alleviated later with some work, either change what is stored in the replay by editing the observations saved or write a converter). I do this by running a game in kaggle like so

```
kaggle-environments run --environment lux_ai_2021 --agents simple_agent simple_agent --render '{"mode": "json"}' --out out.json --debug=True
```

Then upload the out.json to https://2021vis.lux-ai.org/ and scroll to the final turn.

Then using our own engine (which you install via `npm i -g install @lux-ai/2021-challenge`), and download the simple kit from https://github.com/Lux-AI-Challenge/Lux-Design-2021/tree/master/kits/python (or another language)

and run

```
lux-ai-2021 path/to/simple/main.py path/to/simple/main.py --out=replay.json
```

upload to replay viewer again and compare the final state with the other kaggle produced match.

Reply:
Can you add those instructions to that file?

Reply:
yup!